src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodExtraThrows.java

```diff
@@ -0,0 +1,46 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmethod;
+
+public class InputJavadocMethodExtraThrows {
+
+    /**
+     * Extra throws in javadoc it is ok.
+     * @param properties some value
+     * @throws IllegalArgumentException when argument is wrong // ok
+     * @throws NullPointerException indicates null was passed // ok
+     */
+    public InputJavadocMethodExtraThrows(String properties) {
+        if (properties.charAt(0) == 0) {
+            throw new IllegalArgumentException("cannot have char with code 0");
+        }
+        // here is NPE possible
+        properties.isEmpty();
+    }
+
+    /**
+     * Extra throws in javadoc it is ok
+     * @param properties some value
+     * @throws java.lang.IllegalArgumentException when argument is wrong // ok
```

Comment:
http://rveach.no-ip.org/checkstyle/regression/reports/276/openjdk10/index.html#A56
http://rveach.no-ip.org/checkstyle/regression/reports/276/openjdk10/xref/src/java.base/share/classes/jdk/internal/org/xml/sax/EntityResolver.java.html#L139
Original regression said this would be a new violation. Did something change that I missed?

Reply:
```
132      * @exception java.io.IOException A Java-specific IO exception,
133      *            possibly the result of creating a new InputStream
134      *            or Reader for the InputSource.
135      * @see org.xml.sax.InputSource
136      */
137     public abstract InputSource resolveEntity (String publicId,
138                                                String systemId)
139         throws SAXException, IOException;
```

yes but it is case when method throws  and javadoc throws mean the same by one is using shohrt name and another use fully qualified name. 
it is covered by new method `isClassNamesSame` (not very robust implementation, but it is ok), so no violation anymore, as classes are the same.